lib/omniauth/strategies/twitter_ads.rb

```diff
@@ -2,8 +2,8 @@
 
 module OmniAuth
   module Strategies
-    class Twitter < OmniAuth::Strategies::Twitter
-      option :name, 'twitter'
+    class TwitterAds < OmniAuth::Strategies::Twitter
+      option :name, 'twitter_ads'
```

Comment:
@hardylim maybe we could delete this `twitter.rb` file. and only keep the twitter_organic.rb. Just like the facebook_pages.rb here.